ngi_reports/common/ign_sample_report.py

```diff
@@ -363,34 +370,39 @@ def make_plots(self):
 
 
 
-    def check_fields(self):
+    def check_project_fields(self):
         """ Check that the object has all required fields. Returns True / False.
         """
         report_fields = []
         project_fields = ['id', 'sequencing_centre', 'sequencing_platform', 'ref_genome']
-        sample_fields = ['total_reads',  'percent_aligned', 'aligned_reads', 'median_insert_size',
-            'automsomal_coverage', 'ref_above_30X', 'percent_gc']
-        plot_fields = ['coverage_plot', 'cov_frac_plot', 'insert_size_plot', 'gc_dist_plot', 'snpEFf_plot']
 
         for f in report_fields:
-            if f not in self.info.keys():
-                self.LOG.error('Mandatory report field missing: '+f)
+            if not self.info.has_key(f):
+                self.LOG.error('Mandatory report field missing: {}'.format(f))
                 return False
         for f in project_fields:
-            if f not in self.project.keys():
+            if not self.project.has_key(f):
                 import json
                 print(json.dumps(self.project, indent=4))
```

Comment:
Yup, now looks like this:
```
2015-05-22 11:39:19,242 - NGI Reports - ERROR - Mandatory sample-level field(s) missing for P1457_102: total_reads, percent_aligned, aligned_reads, median_insert_size, automsomal_coverage, ref_above_30X, percent_gc, coverage_plot, cov_frac_plot, insert_size_plot, gc_dist_plot, snpEFf_plot
```
I think the `json` thing was just so that I could see what values were being assigned to each key in the dict, a quick way to give nice readable layout.